back/app/services/idea_custom_fields_service.rb

```diff
@@ -206,5 +208,52 @@ def section1_title?(field, attribute)
     field.code == 'ideation_section1' && attribute == :title_multiloc
   end
 
+  def add_user_fields(fields)
+    return fields unless @participation_method.supports_user_fields_in_form?
+
+    fields = fields.to_a # sometimes array passed in, sometimes active record relations
+
+    last_page = fields.last
+    fields.pop # Remove the last page so we can add it back later
```

Comment:
Another suggestion if we keep it like this way:
```suggestion
    last_page = fields.pop # Remove the last page so we can add it back later
```